File: cpp_classes/card.cpp
#include "card.hpp"

using namespace std;

Card::Card (string sign) {
	this->sign = sign;
	this->reversed = false;
}
		
string Card::getSign() {
	return this->sign;
}

void Card::setSign(string sign) {
	this->sign = sign;
}

bool Card::isReversed() {
	return reversed;
}

void Card::reverse(bool reversed) {
	this->reversed = reversed;
}

File: gui/src/main/java/Card.java
public class Card {
    private int sign;
    private boolean reversed;

    public Card (int sign) {
        this.sign = sign;
        this.reversed = false;
    }


    public int getSign() {
        return sign;
    }

    public void setSign(int sign) {
        this.sign = sign;
    }

    public boolean isReversed() {
        return reversed;
    }

    public void setReversed(boolean reversed) {
        this.reversed = reversed;
    }
}